nle/env/tasks.py

```diff
@@ -194,7 +196,7 @@ def __init__(self, *args, **kwargs):
 
         super().__init__(*args, options=options, **kwargs)
 
-    def _reward_fn(self, last_observation, observation, end_status):
+    def _reward_fn(self, last_observation, action, observation, end_status):
         """Difference between previous gold and new gold."""
         del end_status  # Unused
```

Comment:
Thanks! To keep this in line with how this code was written so far, would you mind adding `del action` lines to make linters happy about `action` not being used in these functions at this point?

Reply:
Sure thing, I've made that change